### PR TITLE
fix: wait more time for Window :cry:

### PR DESCRIPTION
**Pull request by popomore**

### CODE REVIEW

File: lib/cov_command.js [hunk 1]
```diff
@@ -44,8 +44,8 @@ class CovCommand extends Command {
     yield this.helper.forkNode(covFile, covArgs, opt);
     rimraf.sync(process.env.TMPDIR);
 
-    // wait 1 second for Windows
-    yield sleep(1000);
+    // wait 5 second for Windows
+    yield sleep(5000);
```
Comment (atian25) [inline lib/cov_command.js]: 判断下是 win 再执行吧?

Comment (popomore) [inline lib/cov_command.js]: 改了

**Comment by atian25** [comment]
+1

**Comment by fengmk2** [comment]
+1

**Comment by fengmk2** [comment]
@atian25 合并发布吧。

**Comment by atian25** [comment]
1.5.3

**Pull request merged**